packages/types/src/localization.ts

```diff
@@ -121,6 +139,10 @@ type _LocalizationResource = {
       subtitle: LocalizationValue;
       actionText: LocalizationValue;
       actionLink: LocalizationValue;
+      useEmailActionLink: LocalizationValue;
```

Comment:
Shouldn't these have the same base name and an extra id after `__` like
```
  actionLink__use_email_username: LocalizationValue;
```

Reply:
Right